delivery/client.py

```diff
@@ -19,12 +19,26 @@
 
 
 logger = logging.getLogger(__name__)
+jwt_key = 'bearer_token'
```

Comment:
nit: use upper case for constants as described in [PEP Style Guide](https://peps.python.org/pep-0008/#constants)